massa-models/src/config/constants.rs

```diff
@@ -240,6 +240,8 @@ pub const POOL_CONTROLLER_DENUNCIATIONS_CHANNEL_SIZE: usize = 1024;
 pub const MAX_GAS_PER_BLOCK: u64 = u32::MAX as u64;
 /// Maximum of GAS allowed for asynchronous messages execution on one slot
 pub const MAX_ASYNC_GAS: u64 = 1_000_000_000;
+/// Gas used by a base operation (transaction, roll buy, roll sell)
+pub const BASE_OPERATION_GAS_COST: u64 = MAX_GAS_PER_BLOCK / (MAX_OPERATIONS_PER_BLOCK as u64);
```

Comment:
use a constant here directly

Reply:
done (but I found it better expressed relatively to other constants)

Reply:
I'm scared of "unexpected correlations. Eg. if we change the max_block_gas we don't expect the base cost of ops to be affected